safe/gui/tools/wizard/step_fc00_functions1.py

```diff
@@ -62,17 +64,28 @@ def get_next_step(self):
         """
         return self.parent.step_fc_functions2
 
-    def selected_functions_1(self):
-        """Obtain functions available for hazard an exposure selected by user.
+    def selected_value(self, layer_purpose_key):
+        """Obtain selected hazard or exposure.
 
-        :returns: List of the available functions metadata.
-        :rtype: list, None
+        :param layer_purpose_key: A layer purpose key, can be hazard or
+            exposure.
+        :type layer_purpose_key: str
+
+        :returns: A selected hazard or exposure definition.
+        :rtype: dict
         """
-        selection = self.tblFunctions1.selectedItems()
-        if len(selection) != 1:
-            return []
+        if layer_purpose_key == layer_purpose_exposure['key']:
+            role = RoleExposure
```

Comment:
Is-it normal that it's not `role_exposure` ? I guess it's legacy, but maybe we can update this.